src/snowcli/cli/object/manager.py

```diff
@@ -16,9 +14,12 @@ def _get_object_names(object_type: str) -> ObjectNames:
 
 
 class ObjectManager(SqlExecutionMixin):
-    def show(self, *, object_type: str, like: str) -> SnowflakeCursor:
+    def show(
+        self, *, object_type: str, like: str | None = None, **kwargs
+    ) -> SnowflakeCursor:
         object_name = _get_object_names(object_type).sf_plural_name
-        return self._execute_query(f"show {object_name} like '{like}'")
+        like = like or "%%"
+        return self._execute_query(f"show {object_name} like '{like}'", **kwargs)
```

Comment:
IMO `ObjectManager` should not reveal its implementation details. Passing `kwargs` makes our "commands" layer dependant not only on `ObjectManager` interface but also on its implementation (interface of `_execute_query` in this case). Wdyt?

Reply:
We should change all executions to use DictCursor, it will make our live easier but this is much bigger scope. Introducing a dedicated function like `_execute_query_dict` will also possibly have a cascade effect. So I decided to do so for now.